src/main/java/com/hms_networks/sc/cumulocity/api/CConnectorMqttMgr.java

```diff
@@ -82,6 +83,9 @@
   /** String containing the ID of the MQTT client. */
   private final String mqttId;
 
+  /** List of child devices which have been registered to Cumulocity. */
+  private final List registeredChildDevices = new ArrayList();
```

Comment:
should there be a getter function for this?

Reply:
No - its private/only accessed from inside this class